mercury/static/mercury/style.css

```diff
@@ -418,15 +418,16 @@ display: block;
   table{
     border-collapse: collapse;
     border: 1px solid black;
-    width: 600px;
-    height: 400px;
-    overflow: 400px;
+    width: 100%;
+    height: 100%;
+    
```

Comment:
This change also changes it for the "Raw Telemetry". Is that intentional?

Reply:
Okay, read your other comment. Maybe we should be using `overflow: auto` according to https://www.w3schools.com/css/css_overflow.asp. What do you think?